packages/core/contracts/oracle/implementation/Proposer.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "./Finder.sol";
+import "./Governor.sol";
+import "./Constants.sol";
+import "./Voting.sol";
+import "./AdminIdentifierLib.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/**
+ * @title Proposer contract that allows anyone to make governance proposals with a bond.
+ */
+contract Proposer is Ownable, Testable {
```

Comment:
the owner of this is intended to be the DVM, yes? it's up to the deployer to transfer this ownership? would it not be better to do the transfer in the constructor?

Reply:
Hmmmm, yeah, I think this makes sense. The only downside I can imagine is that I could imagine us wanting more flexibility in certain deployment scenarios. I can't think of any examples, though, so I'll go ahead and make this change.

Reply:
Done. It made the tests a little trickier to implement since you have to go through the governor to test the bond changing functionality, but it seems to work fine!

Reply:
Hmm I would have leaned to letting deployer transfer ownership, not doing it automatically in constructor. Up to you @mrice32